### DISCUSSION TITLE
minimum intensity of laser

### DISCUSSION THREAD

**Discussion by dpkhatua** [epochpic/epoch#507] [General]
Hii All,
Can anyone inform me about the minimum intensity of laser that can be used in EPOCH to study laser-plasma interaction?

Thank you.
Durga Prasad Khatua.

**Comment by Status-Mirror** [comment]
EPOCH updates the fields using Maxwell's equations (see the [field solver](https://epochpic.github.io/developer/core_structure/field_solver.html) documentation). As a result, EPOCH is only aware of matter which produces a current - and because particles are pushed via the Lorentz force, EPOCH fields can only see the motion of charged particles (a plasma).

You could initialise EPOCH with a neutral gas, and these gas molecules may indeed move, but because their motion generates no current, the EPOCH field-solver will not see them. In this case, the laser would move through the gas as if it were not there. This is also true for neutral solid targets.

So there are two answers to your question:
- Computationally, if you initialise a simulation with a laser and a plasma, the laser should be modelled correctly. However, if you initialise your target with neutral atoms, the laser would not "see" them unless you switch on field-ionisation, and have sufficient intensity to ionise the target.
- Physically, it only makes sense to consider a laser strong enough to ionise the material into a plasma. The laser intensity which produces an EM field comparable to the electric field between electron and nucleus (and is thus capable of ionisation) is around $10^{14} Wcm^{-2}$.

**Reply by dpkhatua** [reply]
Thank you, Stuart.
